CHANGES.md

```diff
@@ -20,6 +20,7 @@ Next (unreleased)
 - Spectrogram plugin: move to separate directory (#1996)
 - Mediasession plugin: move to separate directory (#2020)
 - Timeline plugin: move to separate directory (#2018)
+- Trigger `waveform-ready` event when provided peaks are drawn (#2034)
```

Comment:
can you merge this branch forward and move it to the Next release section?

Reply:
Done.

Reply:
this wasn't updated

Reply:
Apologies, I neglected to push the commit last night. It's present now.